critiquebrainz/frontend/templates/search/results.html

```diff
@@ -101,5 +101,29 @@
       </td>
     </tr>
 
+  {% elif type=="recording" %}
+    <tr>
+      <td>
+        <a href="{{ url_for('recording.entity', id=result.id) }}">{{ result['title'] }}</a>
+      </td>
+      <td>
+        {% if result['length'] %}
+          {{ result['length'] | track_length_ms }}
+        {% else %}
+          -
+        {% endif %}
+      </td>
+      <td>
+        <a href="{{ url_for('artist.entity', mbid=result['artist-credit'][0].artist.id) }}">{{ result['artist-credit'][0]['artist']['name'] or '-' }}</a>
+      </td>
+      <td>
+        {% if result['release-list'] %}
+          <a href="{{ url_for('release.entity', id=result['release-list'][0].id) }}">{{ result['release-list'][0].title or '-' }}</a>
```

Comment:
why are we only showing 1 release? (I agree that this could get problematic with some recordigns). Maybe we could just show the "most correct" release here? (whatever that means)

Reply:
I agree but what should be the "most correct" release?

Reply:
CB-418